CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/). This project attempts to match the major and minor versions of [stactools](https://github.com/stac-utils/stactools) and increments the patch number as needed.
 
+## [v0.3.0] - 2021-09-28
```

Comment:
Not sure if the latest changes are big enough for a `v0.3.0`, should be a `v0.2.1` I think.